README.md

```diff
@@ -37,7 +38,7 @@ $ export SLACKCAT_WEBHOOK_URL=https://hooks.slack.com/services/super/secret/toke
 
 Will override file config.
 
-Optional vars: `SLACKCAT_CHANNEL`, `SLACKCAT_USERNAME`, `SLACKCAT_ICON`
+Optional vars: `SLACKCAT_CHANNEL`, `SLACKCAT_USERNAME`, `SLACKCAT_ICON`, `https_proxy`
 
```

Comment:
Technically, go's "http" clause supports http_proxy, https_proxy, etc natively (like many other linux libraries). In my original testing, had a bad document saying HTTPS_PROXY, which didn't work.

Lowercase http_proxy works without any additional code needed. Figured adding to the readme would remind future users.